docs/setting_region.md

```diff
@@ -0,0 +1,40 @@
+# Setting GCP region
```

Comment:
As discussed offline, lets add info from [this page](https://cloud.google.com/dataflow/docs/concepts/regional-endpoints), to guide users into using the same region as their GCS bucket/BigQuery dataset, if they are not multi-regional.

Reply:
done.